### ISSUE TITLE
Presenter Mode not available in VS2019

### ISSUE THREAD

**Issue by calloncampbell** [Microsoft/VS-PPT#228]
The presenter mode (on, off, edit) is not available in VS2019. It would be nice if this functionality was added/supported for this version of Visual Studio.

In addition I think when going into Presenter Mode, the theme should change to "Light" as the dark themes are not usable when presenting on projectors. Maybe this could be an option to change to Light mode when going into Presenter mode, otherwise leave the currently selected theme.

**Comment by jonathantower** [comment]
I haven't been able to get any of the Quick Launch tasks from the extension to work in Visual Studio 2019.
![2019-05-13 08_26_25-Microsoft Visual Studio](https://user-images.githubusercontent.com/1853927/57621212-dacdc100-7558-11e9-9b2d-03e1efbc9124.png)

**Comment by apwestgarth** [comment]
The only way I have been able to get Presenter Mode to execute in VS2019 is to enable in VS2017 and then relaunch VS2019 but that requires me to have VS2017 installed when I'd really like to just run VS2019

**Comment by calloncampbell** [comment]
I only have VS2019 installed and have no desire to install VS2017. This functionality should just work or be a simple enough fix to make it work in VS2019.

**Comment by apwestgarth** [comment]
@calloncampbell I agree I was just pointing out that I have found a way to work around it on a machine where I still have both installed.

**Comment by Xarkam** [comment]
Hello,
I was also looking for the presentation mode in vs2019.

I discovered that the Viasfora extension contains presentation mode.
This may be a paliative pending a possible fix for the next version of vs2019.

**Comment by calloncampbell** [comment]
This is still not fixed?

**Comment by KrisvanderMast** [comment]
I was also surprised to see this isn't working in VS2019 with the productivity tools installed. Any ETA when this will ship again?

**Comment by adamhill** [comment]
I too would like Presentation Mode brought back

**Comment by calloncampbell** [comment]
I've found this tool and works decently in VS2019, but preferred the Presentation Mode extension.
https://marketplace.visualstudio.com/items?itemName=ABECE-Software-Solutions.Presenter
